plugin/othttp/common.go

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package othttp
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/otel/api/core"
+	"go.opentelemetry.io/otel/api/trace"
+)
+
+// Attribute keys that can be added to a span.
+const (
+	HostKey       = core.Key("http.host")        // the HTTP host (http.Request.Host)
```

Comment:
`key.New()` is an alias for `core.Key()`. Some have suggested we should do something about the `core` package, others think `key` is frivolous. Just a thought.

Reply:
These were just lifted out of the handler.go implementation since they're now common to both the handler and the transport.  Happy to change them if we're going to make a wholesale move away from `core.Key()`, but if it's just rearranging furniture could we do it in a separate PR?

Reply:
Sorry, I didn't realize at first these weren't new.